assets/scripts/ui/RangeSlider.scss

```diff
@@ -64,7 +64,7 @@ $range-track-height: 6px !default;
       &::-webkit-slider-thumb {
         box-shadow: inset 0 0 0 1px white,
           // Match background color
-            0 0 0 1px white,
+          0 0 0 1px white,
```

Comment:
This is really strange, my version of Prettier complains if that one line indent _isn't_ there. Is that not true for you?

It seems this should be the correct fix, though.

Reply:
It kept going back and forth for me, which I also thought was weird.

Reply:
We may have to rewrite this different. Does it complain if the comment (line 66) is moved inline to the end of line 67?

Reply:
@louh for me Prettier auto formatted that to: 
```
      &::-webkit-slider-thumb {
        box-shadow: inset 0 0 0 1px white, 0 0 0 1px white,
          // Match background color
          0 0 0 3px $colour-emerald-400;
      }
```

Which, I think, is definitely worse. :-D I think the way I have it on this branch makes the most sense, so maybe its more of an issue of troubleshooting why your local prettier is clashing?

Reply:
Ideally this is what it would look like:

```css
      &::-webkit-slider-thumb {
        /* prettier-ignore */
        box-shadow:
          inset 0 0 0 1px white,
          0 0 0 1px white, /* Match background color */
          0 0 0 3px $colour-emerald-400;
      }
```

Prettier seems to have a problem with this, https://github.com/prettier/prettier/issues/7039 (still open) and compounded by the fact that we have a comment splitting the lines.

I am going to `/* prettier-ignore */` this section for now and document the rationale.